org.eclipse.xtext/src/org/eclipse/xtext/build/IncrementalBuilder.java

```diff
@@ -286,6 +285,18 @@ protected void unloadResource(URI uri) {
 			return new IncrementalBuilder.Result(request.getState(), resolvedDeltas);
 		}
 
+		/**
+	 	* Return a new Serializable resource description from a {@link IResourceDescription}.
+	 	*
+	 	* @param description
+	 	*            the serializable description, must not be {@code null}
+	 	* @return the new serializable resource description, never {@code null}
+	 	* @since 2.26
+	 	*/
+		protected IResourceDescription getSerializableResourceDescription(IResourceDescription description) {
```

Comment:
can you describe your usecase

Reply:
Hi @cdietrich , sorry, I do not why the my initial description did not make it to the PR. I have added it 5 min ago. Do you see it?

Reply:
now i see it, before it was not visible